python/taichi/examples/simulation/implicit_fem.py

```diff
@@ -0,0 +1,353 @@
+import argparse
+
+import numpy as np
+from taichi._lib import core as _ti_core
+
+import taichi as ti
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--exp', default='implicit')
+#parser.add_argument('--exp', default='explicit')
```

Comment:
ditto

Reply:
User can comment line 9 and uncomment line 10 to switch to the explicit version, so I think this line can be kept here.
Same for line 13 and 14.

Reply:
FYI you can use `choices`: https://docs.python.org/3/library/argparse.html#choices

Reply:
I will use `choices`.